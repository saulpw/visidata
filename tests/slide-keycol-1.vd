sheet	col	row	longname	input	keystrokes	comment
			open-file	sample_data/sample.tsv	o	
sample	OrderDate		key-col		!	toggle current column as a key column
sample	Region		key-col		!	toggle current column as a key column
sample	Rep		key-col		!	toggle current column as a key column
sample	Rep		slide-leftmost		gH	slide current column all the way to the left of sheet
