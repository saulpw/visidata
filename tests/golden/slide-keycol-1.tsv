Rep	OrderDate	Region	Item	Units	Unit_Cost	Total
Jones	2016-01-06	East	Pencil	95	1.99	189.05
Kivell	2016-01-23	Central	Binder	50	19.99	999.50
Jardine	2016-02-09	Central	Pencil	36	4.99	179.64
Gill	2016-02-26	Central	Pen	27	19.99	539.73
Sorvino	2016-03-15	West	Pencil	56	2.99	167.44
Jones	2016-04-01	East	Binder	60	4.99	299.40
Andrews	2016-04-18	Central	Pencil	75	1.99	149.25
Jardine	2016-05-05	Central	Pencil	90	4.99	449.10
Thompson	2016-05-22	West	Pencil	32	1.99	63.68
Jones	2016-06-08	East	Binder	60	8.99	539.40
Morgan	2016-06-25	Central	Pencil	90	4.99	449.10
Howard	2016-07-12	East	Binder	29	1.99	57.71
Parent	2016-07-29	East	Binder	81	19.99	1619.19
Jones	2016-08-15	East	Pencil	35	4.99	174.65
Smith	2016-09-01	Central	Desk	2	125.00	250.00
Jones	2016-09-18	East	Pen Set	16	15.99	255.84
Morgan	2016-10-05	Central	Binder	28	8.99	251.72
Jones	2016-10-22	East	Pen	64	8.99	575.36
Parent	2016-11-08	East	Pen	15	19.99	299.85
Kivell	2016-11-25	Central	Pen Set	96	4.99	479.04
Smith	2016-12-12	Central	Pencil	67	1.29	86.43
Parent	2016-12-29	East	Pen Set	74	15.99	1183.26
Gill	2017-01-15	Central	Binder	46	8.99	413.54
Smith	2017-02-01	Central	Binder	87	15.00	1305.00
Jones	2017-02-18	East	Binder	4	4.99	19.96
Sorvino	2017-03-07	West	Binder	7	19.99	139.93
Jardine	2017-03-24	Central	Pen Set	50	4.99	249.50
Andrews	2017-04-10	Central	Pencil	66	1.99	131.34
Howard	2017-04-27	East	Pen	96	4.99	479.04
Gill	2017-05-14	Central	Pencil	53	1.29	68.37
Gill	2017-05-31	Central	Binder	80	8.99	719.20
Kivell	2017-06-17	Central	Desk	5	125.00	625.00
Jones	2017-07-04	East	Pen Set	62	4.99	309.38
Morgan	2017-07-21	Central	Pen Set	55	12.49	686.95
Kivell	2017-08-07	Central	Pen Set	42	23.95	1005.90
Sorvino	2017-08-24	West	Desk	3	275.00	825.00
Gill	2017-09-10	Central	Pencil	7	1.29	9.03
Sorvino	2017-09-27	West	Pen	76	1.99	151.24
Thompson	2017-10-14	West	Binder	57	19.99	1139.43
Andrews	2017-10-31	Central	Pencil	14	1.29	18.06
Jardine	2017-11-17	Central	Binder	11	4.99	54.89
Jardine	2017-12-04	Central	Binder	94	19.99	1879.06
Andrews	2017-12-21	Central	Binder	28	4.99	139.72
